scripts/lunch.coffee

```diff
@@ -0,0 +1,66 @@
+# Description:
+#   Lunch recording technology.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   <botname> lunch at [location]
+#   <botname> lunch me
+
+makeRecord = (now = new Date()) ->
+
+  return {
+    last_lunch: now.getFullYear() + "-" + (now.getMonth() + 1) + "-" + now.getDate()
+    timestamp: now.getTime()
+  }
+
+module.exports = (robot) ->
+
+  robot.respond /lunch at (.+)$/i, (msg) ->
+
+    location = msg.match[1]
+    robot.brain.data.lunch_spots ||= {}
+
+    if record = robot.brain.data.lunch_spots[location]
+      msg.send "last lunch at #{location} was #{record.last_lunch}"
```

Comment:
I feel like it's non-obvious that it "worked" in this case. Should probably be like:

> pgbot lunch at chipotle
> OK, enjoy lunch at chipotle! You were there last on 1-12-2015

And...

> pgbot lunch at eleven madison park
> OK, eleven madison park for lunch! That looks like a new place - hope it's good...